examples/profiling_voice.py

```diff
@@ -0,0 +1,26 @@
+# +
+import torch
+import torch.tensor as T
+
+from torchsynth.globals import SynthGlobals
+from torchsynth.synth import Voice
+
+# %load_ext autoreload
+# %autoreload 2
+# -
+
```

Comment:
Can we remove this until it's a little more mature?

Reply:
Oh ya totally